### PR TITLE
Ember CLI 2016-10-06

### PR DESCRIPTION
**Pull request by nathanhammond**
Reactions for approval, comments for corrections.

/cc @chadhietala @Turbo87 @hjdivad @kellyselden
/bcc @trabus @kategengler @rwjblue @stefanpenner

### CODE REVIEW

File: ember-cli/2016-10/october-06.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+- [We're not going to change our messaging from the blog post.](http://emberjs.com/blog/2016/09/07/ember-node-lts-support.html#toc_node-js-0-10-node-js-0-12)
+
+> The first version which will not support Node.js 0.10 will be Ember CLI 2.9.0 scheduled to be released on October 17.
+
+- We imply heavily via that quote that 2.9 will not support 0.10, but there is a hidden conditional a few sentences before.
+- Unlikely people will complain about supporting it longer.
+
+### npm v4
+
+- Fortunately just landed v3.
```
Comment (stefanpenner) [inline ember-cli/2016-10/october-06.md]: good work!

File: ember-cli/2016-10/october-06.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+- We should make sure we're not obviously broken by it.
+
+### Travis + Node.js 0.12 + npm
+
+- Tons of errors, breaking.
```
Comment (stefanpenner) [inline ember-cli/2016-10/october-06.md]: my recommendation is to move to "allowed failures" until travis fixes. Or we consider an alternative installation strat for node 0.12

Comment (Turbo87) [inline ember-cli/2016-10/october-06.md]: seems to be somewhat mitigated by https://github.com/ember-cli/ember-cli/pull/6317, which is a useful thing to do for network installs anyway

Comment (stefanpenner) [inline ember-cli/2016-10/october-06.md]: https://github.com/ember-cli/ember-cli/pull/6327 should get us out of hot water (we did the same thing for a period where node 5 was broke)

**Pull request merged**